framework/src/media/audio/audio_manager.c

```diff
@@ -1614,8 +1614,22 @@ audio_manager_result_t find_stream_in_device_with_process_type(device_process_ty
 	audio_card_info_t *card;
 
 	if (type == AUDIO_DEVICE_PROCESS_TYPE_NONE) {
-		meddbg("Type is AUDIO_DEVICE_PROCESS_TYPE_NONE\n");
-		return AUDIO_MANAGER_INVALID_PARAM;
+		/* TODO : will be implemented later based on audio subsystem's implemantation */ 
```

Comment:
please change implemantation to implementation

Reply:
Fixed it